README.md

```diff
@@ -359,6 +359,15 @@ For example, if you want to build RPMs using `test/rpm_build` image and override
 RPM_BUILD_IMAGE=test/rpm_build RPM_BUILD_OPTIONS=my_options bin/build -d images -r manageiq -b
 ```
 
+An example options.yml to use "feature" branch on the "example" fork of ManageIQ/manageiq:
+
+```
+repos:
+  manageiq:
+    url:  https://github.com/example/manageiq.git
+    ref:  feature
```

Comment:
For reference, I tried a git ref, a sha1, here and it doesn't work.  It needs to be a branch.  I think a tag might also fail.

Reply:
Added `must be tag or branch`. We use tag for release builds, so tag works 😄